deeds-dapp-webapp/src/main/webapp/skin/less/deeds.less

```diff
@@ -238,6 +239,24 @@ h4 {
 .primary-border-color {
   border: 1px solid #3f8487 !important;
 }
+
+.dark-grey-border-color {
+  border: 1px solid @darkGrey !important;
+}
+
+.logo-m {
+  filter: invert(47%) sepia(46%) saturate(419%) hue-rotate(134deg) brightness(89%) contrast(90%);
+  opacity: 0.5;
+  position: absolute;
+  right: -80px;
+}
+
+.primary-btn:hover {
```

Comment:
It would better to use buttons with style `<v-btn color="primary">...</v-btn>`

Reply:
should we inherit the same style as Vuetify style of the Meeds package by including the file `vuetify-all.less` and replace the one defined in dApp ? WDYT ?

Reply:
I added in a commit the `vuetify-all.less` inherited from Meeds Package as suggested.